app/views/shared/_playbook_options.html.haml

```diff
@@ -6,7 +6,7 @@
     = _("Playbook Catalog Item")
   .col-md-8
     = select_tag('service_template_id',
-    options_for_select([["<#{_('Choose')}>", nil]] + @edit[:ansible_playbooks],
+    options_for_select([["<#{_('Choose')}>", nil]] + @edit[:ansible_playbooks].map{|request, index| [_(request), index]},
```

Comment:
What are we trying to translate here? Ansible playbook names? (user-defined content)